package/MDAnalysis/core/universe.py

```diff
@@ -1613,7 +1619,10 @@ def guess_TopologyAttrs(
         # in the same order that the user provided
         total_guess = list(dict.fromkeys(total_guess))
 
-        objects = ['bonds', 'angles', 'dihedrals', 'impropers']
+        objects = set(
```

Comment:
I keep having to rewind to check what `object` is, which probably means it's poorly named - ideally we should rename, but in the spirit of "avoiding further changes", maybe let's add a comment on what we're doing here?

```suggestion
        # Set of all Connectivity related attribute names
        # used to special case attribute replacement after calling the guesser
        objects = set(
```